cmd/server/README.md

```diff
@@ -20,6 +21,7 @@ configuring the following environment variables:
 | THREESCALE_CACHE_REFRESH_RETRIES | Sets the number of times unreachable hosts will be retried during a cache update loop              | 1       |
 | THREESCALE_ALLOW_INSECURE_CONN   | Allow to skip certificate verification when calling 3scale API's. Enabling is not recommended      | false   |
 | THREESCALE_CLIENT_TIMEOUT_SECONDS| Sets the number of seconds to wait before terminating requests to 3scale System and Backend        | 10      |
+| THREESCALE_GRPC_CONN_MAX_MINUTES | Sets the maximum amount of time (+/-10% jitter) a connection may exist before it will be closed    | 1       |
```

Comment:
maximum amount of time **in minutes** (+/-10% jitter) - Alternatively, I think allowing seconds here is just fine and allows for better control, what do you think?